os/win/dnd.h

```diff
@@ -0,0 +1,96 @@
+// LAF OS Library
+// Copyright (C) 2024  Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#ifndef OS_WIN_DND_H_INCLUDED
+#define OS_WIN_DND_H_INCLUDED
+#pragma once
+
+#include "os/dnd.h"
+
+#include <ole2.h>
+
+namespace os {
+
+  // COM object pointer wrapper to simplify reference counting.
+  template<class T>
+  class ComPtr {
```

Comment:
We have a COM pointer wrapper in `base::ComPtr` (`#include "base/win/comptr.h"`)